openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/ActivityFeedTab.component.tsx

```diff
@@ -168,23 +162,38 @@ export const ActivityFeedTab = ({
       )
     );
     setActiveThread();
+    setIsFullWidth(false);
   };
 
   const placeholderText = useMemo(() => {
     if (activeTab === ActivityFeedTabs.ALL) {
       return (
-        <Transi18next
-          i18nKey="message.no-activity-feed"
-          renderElement={<Link to={ROUTES.EXPLORE} />}
-          values={{
-            explored: t('message.have-not-explored-yet'),
-          }}
-        />
```

Comment:
This was needed for Landing page widget can you confirm if we haven't removed it

Reply:
In Landing Page for displaying the Activity Feed we are using <ActivityFeedListV1 /> component which is being rendered from FeedsWidget component . Both of these original components are unaltered, so this change won't be affecting the landing page